src/types.ts

```diff
@@ -164,6 +175,7 @@ export type VenueDetails<T = TranslationsObject> = {
   telephone: T | null;
   ontologyTree: Ontology[];
   ontologyWords: Ontology[];
+  accessibilitySentences: AccessibilityTranslationsObject;
```

Comment:
Is this type definition correct? The generics use with this type allows for it to be used to represent a value that contains all the translations, as well as a value where a translation has been picked (`VenueDetails<T = TranslationsObject>`). Will the type of `accessibilitySentences` always be of type `AccessibilityTranslationsObject`?

Reply:
Ahh, seems that I misunderstood the type definition. In that case the accessibility sentences should be `AccessibilityTranslationsObject | AccessibilitySentences`